CUBIC PM2005/PM2105 Laser Particle Sensor Module (#4696)
OttoWinter/esphomedocs: components/sensor/pm2005.rst

CUBIC PM2005/PM2105 Laser Particle Sensor Module
================================================

.. seo::
    :description: Instructions for setting up PM2005/PM2105 Particulate matter sensor
    :image: pm2005.png
    :keywords: pm2005, pm2105, DAIKIN BRY88AB151K, BRY88AB151K

The ``pm2005`` sensor platform allows you to use your PM2005/PM2105 laser particulate matter sensor with ESPHome.

.. figure:: images/pm2005.png
    :align: center
    :width: 50.0%

    CUBIC PM2005/PM2105 Laser Particle Sensor Module.

> The sensor communicate with board by :ref:`I²C <i2c>` protocol.

Reviewer: ```suggestion
The sensor communicates with the board via the :ref:`I²C <i2c>` protocol.
```

.. code-block:: yaml

    # Example configuration entry

>     i2c:

Reviewer: ```suggestion
    i2c:
      sda: GPIOXX
      scl: GPIOXX
```

    sensor:

>       - platform: pm2005

Reviewer: ```suggestion
      - platform: pm2005
        update_interval: 60s
```

        pm_1_0:
          name: "PM1.0"
        pm_2_5:
          name: "PM2.5"
        pm_10_0:
          name: "PM10.0"

Configuration variables:
------------------------

> - **type** (*Optional*): ``pm2005`` (default) or ``pm2105``

Reviewer: ```suggestion
- **type** (*Optional*): ``pm2005`` (default) or ``pm2105``

- **update_interval** (*Optional*, :ref:`config-time`): The interval to check the sensor. Defaults to ``60s``.
```

CUBIC PM2005/PM2105 Laser Particle Sensor Module
================================================

.. seo::
    :description: Instructions for setting up PM2005/PM2105 Particulate matter sensor
    :image: pm2005.png
    :keywords: pm2005, pm2105, DAIKIN BRY88AB151K, BRY88AB151K

The ``pm2005`` sensor platform allows you to use your PM2005/PM2105 laser particulate matter sensor with ESPHome.

.. figure:: images/pm2005.png
    :align: center
    :width: 50.0%

    CUBIC PM2005/PM2105 Laser Particle Sensor Module.

The sensor communicates with the board via the :ref:`I²C <i2c>` protocol.

.. code-block:: yaml

    # Example configuration entry

>     i2c:
>       sda: GPIOXX
>       scl: GPIOXX
> 

Reviewer 2: We do not include configurations of other components here
```suggestion
```
